.circleci/config.yml

```diff
@@ -260,13 +352,34 @@ workflows:
               only:
                 - main
     jobs:
-      - job_e2e-test:
+      - job_e2e-test-linux:
+          matrix:
+            parameters:
+              package_version: [ "latest" ]
+          post-steps:
+            - run:
+                name: Send Slack notification
+                command: |
```

Comment:
Convert this to a `cmd_` and instead have all of these tests run that command on the `post-steps` - this way you don't have duplicate the curl command